tests/acceptance/ignore-logs/commands.yaml

```diff
@@ -0,0 +1,21 @@
+steps:
```

Comment:
Maybe note that the agent output in this case is a little wonky ("looking at 2 changed paths" and "looking for current issues in 2 files" but then ignoring those files)